Bug boo#958462

Allow the smart bash builtin completion if normal completion scripting
does not return anything.

---
 bash_completion |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- bash_completion
+++ bash_completion	2017-07-04 07:48:24.632925756 +0000
@@ -1945,11 +1945,13 @@ _longopt()
 complete -F _longopt -o filenames a2ps awk base64 bash bc bison cat chroot colordiff cp \
     csplit cut date df diff dir du enscript expand fmt fold gperf \
     grep grub head irb ld ldd less ln m4 md5sum mkdir mkfifo mknod \
-    mv nl nm objcopy objdump od paste pr ptx readelf rm rmdir \
+    mv nl nm objcopy objdump od paste pr ptx readelf \
     sed sha{,1,224,256,384,512}sum shar sort split strip sum tac tail tee \
     texindex touch tr unexpand uniq vdir wc who
 complete -F _longopt -o default env netstat seq uname units
 complete -F _longopt -o bashdefault -o default -o filenames -o nospace ls ll la l ls-l lf
+complete -F _longopt -o bashdefault -o default -o filenames -o nospace rm rmdir
+
 
 declare -A _xspecs
 _filedir_xspec()
